scripts/util/sqa.py

```diff
@@ -384,6 +425,63 @@ def _create_addon(version, variables) -> Addon:
 
     return addons[0]
 
+
+def create_build(variables) -> Build:
+    with tempfile.TemporaryDirectory(dir=Path.home(), delete=False) as temp_dir:
+        # the name of the addon dir must be 'addon'
+        addon_dir = Path(temp_dir) / "addon"
+        config_dir = addon_dir / "config"
+        config_dir.mkdir(parents=True, exist_ok=True)
+
+        log.info(f"addon directory created at: {addon_dir}")
+
+        env = Environment(
+            loader=FileSystemLoader("scripts/templates/canonical_k8s_sqa_addon"),
+            autoescape=select_autoescape(),
+        )
+        template_files = env.list_templates(extensions="j2")
+
+        for template_name in template_files:
+            template = env.get_template(template_name)
+            rendered = template.render(variables)
+
+            output_filename = Path(template_name).stem
+            output_path = config_dir / output_filename
+            output_path.write_text(rendered)
+
+        cmd = f"build add --deployment-branch solutionsqa/fkb/sku/master-canonicalk8s-jammy-cos --addon {addon_dir} --format json"
```

Comment:
i think you can use wokeignore

```suggestion
       # wokeignore:rule=master
        cmd = f"build add --deployment-branch solutionsqa/fkb/sku/master-canonicalk8s-jammy-cos --addon {addon_dir} --format json"
```